synapse/lib/stormlib/cortex.py

```diff
@@ -1317,3 +1318,47 @@ async def execStormCmd(self, runt, genr):
         await self.runt.printf(f'Storm Pool URL: {url}')
         await self.runt.printf(f'Sync Timeout (secs): {opts.get("timeout:sync")}')
         await self.runt.printf(f'Connection Timeout (secs): {opts.get("timeout:connection")}')
+
+@s_stormtypes.registry.registerLib
+class CortexApi(s_stormtypes.Lib):
+    '''
+    Library for interacting with the Cortex API.
+    '''
+    _storm_locals = (
+        {'name': 'getIdenByNid', 'desc': 'Get the iden for a node by its node id in this Cortex.',
+         'type': {'type': 'function', '_funcname': 'getIdenByNid',
+                  'args': (
+                      {'name': 'nid', 'type': 'int', 'desc': 'The node id of the node.'},
+                  ),
+                  'returns': {'type': 'str', 'desc': 'The iden of the node or None if the node id is not found.'}}},
+        {'name': 'getNidByIden', 'desc': 'Get the node id for an iden in this Cortex.',
+         'type': {'type': 'function', '_funcname': 'getNidByIden',
+                  'args': (
+                      {'name': 'iden', 'type': 'int', 'desc': 'The iden of the node.'},
```

Comment:
are the types backwards here?

Reply:
yeah i flipped them and forgot to switch the types, will fix